### ISSUE TITLE
Nil Error

### ISSUE THREAD

**Issue by jfeaver** [code-and-effect/effective_posts#3]
In Student Energy, a blog post exists which at some point uses the `truncate_html` helper and runs into a Nil error.  The problem begins to form on this line: https://github.com/code-and-effect/effective_posts/blob/8d803f105defe649619528ad5bf75568a528cec2/app/helpers/effective_truncate_html_helper.rb#L27.  Basically, in this HTML there is only one element and it is an `a` element so the `#find` returns `nil`.  I'm not sure what the best fix here is.  @andie-chernysh, you last changed this file and I imagine you would know what the best fix is.  If you have some time, could you investigate please?

**Comment by jfeaver** [comment]
Here's the error:

```
ActionView::Template::Error (undefined method `add_next_sibling' for nil:NilClass):
     6:     %meta{:charset => 'utf-8'}
     7:     %meta{:name => 'viewport', :content => 'width=device-width, initial-scale=1, maximum-scale=1'}
     8:
     9:     = facebook_og_tags
    10:
    11:     = stylesheet_link_tag 'application', :media => 'screen', 'data-turbolinks-track' => true
    12:     = javascript_include_tag '//ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js'
  app/helpers/application_helper.rb:59:in `block in facebook_og_tags'
  app/helpers/application_helper.rb:41:in `tap'
  app/helpers/application_helper.rb:41:in `facebook_og_tags'
  app/views/layouts/application.html.haml:9:in `_app_views_layouts_application_html_haml___3832458843008651609_70220752315140'
```

Here's a relevant pry session:

```
    32:         last_node.add_next_sibling(read_more_node)
    33:       end
    34:
    35:       if ellipsis.present?
    36:         ellipsis_node = Nokogiri::XML::Text.new(ellipsis.to_s, doc)
 => 37:         binding.pry unless last_node
    38:         last_node.add_next_sibling(ellipsis_node)
    39:       end
    40:
    41:       truncated_node.inner_html
    42:     else

[1] pry(#<#<Class:0x007fbb16e37b18>>)> last_node
=> nil
[2] pry(#<#<Class:0x007fbb16e37b18>>)> actual_length
=> 197
[3] pry(#<#<Class:0x007fbb16e37b18>>)> doc
=> #(DocumentFragment:0x3fdd904847b8 {
  name = "#document-fragment",
  children = [
    #(Text "\n"),
    #(Element:0x3fdd904841c8 {
      name = "p",
      children = [
        #(Element:0x3fdd90489b8c {
          name = "a",
          attributes = [
            #(Attr:0x3fdd905380c4 {
              name = "href",
              value = "http://www.ises2015.com"
              })],
          children = [ #(Text "www.ises2015.com")]
          })]
      })]
  })
```

**Comment by matt-riemer** [comment]
Resolved in 0.3.0

**Comment by andie-chernysh** [comment]
@matt-riemer So, I guess, my help is not need now since that was fixed in 0.3.0? I think, I should just bump up effective_posts version in SE Gemfile.
